tests/pipert/core/test_shared_memory.py

```diff
@@ -0,0 +1,43 @@
+import pipert.core.multiprocessing_shared_memory as sm
+
+
+class DummySharedMemoryGenerator(sm.MpSharedMemoryGenerator):
+    def __init__(self):
+        super().__init__("dummy_component", max_count=5)
+
+
+def test_cleanup():
```

Comment:
Is this checking if we really cleaned the memory? Can we check it?

Reply:
There isn't really a way to check it, but the shared memory is removed from the dictionary only after it closes and unlinks, so we can assume that it has been cleaned.